compiler-rt/lib/asan/asan_linux.cpp

```diff
@@ -51,11 +51,9 @@ extern "C" void *_DYNAMIC;
 #  elif SANITIZER_NETBSD
```

Comment:
There is a remaining _DYNAMIC above that should presumably be removed as well (for Android etc)

Reply:
Thx. removed.